src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -1118,4 +1118,7 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
     <value>Platform version is not present for one or more target frameworks, even though they have specified a platform: {0}</value>
     <comment>0 - comma-separated list of TFMs missing a platform version</comment>
   </data>
+  <data name="Log_SkippingPackagesLockFileGeneration" xml:space="preserve">
+    <value>Skipping the lock file regeneration for '{0}'.</value>
```

Comment:
Don't you need a comment to define {0} for localization? Also, I just had a case where they didn't receive any of my comments... :(

Reply:
No, not really. The comment is from the localization team/engineering team, it has no effect on the formatting.

Reply:
Let me rephrase this... the localization team needs to know what your {0} means. I don't see where you have defined it using a comment.

Reply:
`the localization team needs to know what your {0} means.`

Wasn't aware of that. Many of the strings in this file do not have a comment, so I've never though otherwise. 
Especially with sentences that have only 1 format prop.